internal/pkg/cli/flag.go

```diff
@@ -278,6 +279,8 @@ Must be of the format '<keyName>:<dataType>'.`
 	storageNoLSIFlagDescription     = `Optional. Don't ask about configuring alternate sort keys.`
 	storageLSIConfigFlagDescription = `Optional. Attribute to use as an alternate sort key. May be specified up to 5 times.
 Must be of the format '<keyName>:<dataType>'.`
+	storageRDSServerlessVersionFlagDescription = `Optional. Aurora Serverless version. Must be either "v1" or "v2".
+(default v2)`
```

Comment:
If we do decide to set the default in `cmd.Flags().StringVar(&vars.rdsServerlessVersion, storageRDSServerlessVersionFlag, "", storageRDSServerlessVersionFlagDescription)` (in `storage_init.go`), then the `cobra` will automatically append `(default "v2")` to the end of the flag description I think!

 ```suggestion
	storageRDSServerlessVersionFlagDescription = `Optional. Aurora Serverless version. Must be either "v1" or "v2".`
```

Reply:
I set the default value in `cmd.Flags().StringVar(&vars.rdsServerlessVersion, storageRDSServerlessVersionFlag, defaultServerlessVersion, storageRDSServerlessVersionFlagDescription)` and `cobra` appended `(default "v2")` as expected.